packages/evm/test/customCrypto.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Chain, Common } from '@ethereumjs/common'
+import { assert, describe, it } from 'vitest'
+
+import { EVM } from '../src/evm.js'
+import { getActivePrecompiles } from '../src/index.js'
+
+describe('custom crypto', () => {
+  it('should use custom sha256 function', async () => {
+    const customSha256 = (msg: Uint8Array) => {
```

Comment:
Nit: there is no similar test for `ecrecover`

Reply:
Just added.